draft-tsai-duration-00.xml

```diff
@@ -253,7 +245,7 @@ This example must instead be expressed as PT0.123S.</t>
 <sourcecode>PT1.S
 PT1.000S</sourcecode>
 
-<t>Unnecessary decimal sign or fractional digits must be absent.
+<t>A decimal point with no fractional digits is invalid.
```

Comment:
This rewrite loses the fact that trailing zeros is invalid.

Reply:
Perhaps: "A non-zero fractional component is invalid."

Reply:
hold on, must have forgotten to save.

Reply:
> Perhaps: "A non-zero fractional component is invalid."

I must be having a stupidity attack today. I've read that four times and I can't parse it.  
Consider PT4.3S The fractional component is ".3" and it's non-zero. What am I missing?

Reply:
I apologize. I was the one with the brain fart. I meant to say:
> A non-empty, but zero fractional component is invalid.

But that's harder to understand than what you currently have.